resize-event/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="A resize event demo on MDN" />
```

Comment:
Not sure which of these we need. From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta it looks like we don't need `charset` or `http-equiv` anyway.